test/data/testinit.js

```diff
@@ -48,6 +49,32 @@ QUnit.assert.t = function( a, b, c ) {
 	}
 
 	this.deepEqual( f, q.apply( q, c ), a + " (" + b + ")" );
+}
+
+/**
+ * Asserts that a select matches the given IDs.
+ * The select is not bound by a context.
+ * @param {String} a - Assertion name
+ * @param {String} b - Sizzle selector
+ * @param {String} c - Array of ids to construct what is expected
+ * @example t("Check for something", "//[a]", ["foo", "bar"]);
+ * @result returns true if "//[a]" return two elements with the IDs 'foo' and 'bar'
+ */
+QUnit.assert.t = function( a, b, c ) {
+	match( a, b, c, undefined );
+};
+
+/**
+ * Asserts that a select matches the given IDs.
+ * The select is performed with the `#qunit-fixture` context.
+ * @param {String} a - Assertion name
+ * @param {String} b - Sizzle selector
+ * @param {String} c - Array of ids to construct what is expected
+ * @example t("Check for something", "//[a]", ["foo", "bar"]);
+ * @result returns true if "//[a]" return two elements with the IDs 'foo' and 'bar'
+ */
+QUnit.assert.u = function( a, b, c ) {
```

Comment:
It looks to me like `assert.t` does not mean anything in particular, apart from that it's short. 

If we rename `assert.t` into `assert.selectMatchesIds` (or something descriptive like that), the painfully long lines in unit tests will become even more painfully long (i.e. way beyond 80 chars). I would leave the name as is.